modules/broadcast/src/backend/db.ts

```diff
@@ -134,13 +134,12 @@ export default class BroadcastDb {
   }
 
   async getUsersTimezone() {
-    // TODO: need add "timezone" to users
     const attrs = await this.knex('srv_channel_users')
       .select('attributes')
     const timezones = attrs.map(({ attributes: { timezone } }) => timezone)
-    const timezonesMap = [...new Set(timezones)]
+    const timezonesSet = [...new Set(timezones)]
```

Comment:
@dmytropanontko , this isn't a set: it's destructured. You can just run `return [...new Set(timezones)]`